content/docs/components-and-props.md

```diff
@@ -44,11 +44,7 @@ class Welcome extends React.Component {
 
 上述兩種 component 在 React 中是同等的。
 
-<<<<<<< HEAD
 我們將會在[下一個章節](/docs/state-and-lifecycle.html)探討 class 所擁有的額外特性，但在那之前，我們會使用 function component 來保持簡潔。
```

Comment:
Thank you for fixing conflict, can you help update translation?

```suggestion
Function 和 Class component 兩者都擁有額外的特性，我們將會在[下一個章節](/docs/state-and-lifecycle.html)
```

Reply:
@neighborhood999 
Sure! Thanks for your reviewing.